coordinator/tests/services/translation/translation-service.spec.ts

```diff
@@ -44,3 +45,27 @@ test(
     TestResources.examplePrescription1.hl7V3Message
   )
 )
+
+test("convertFhirMessageToHl7V3ParentPrescriptionMessage result has no lower case UUIDs", () => {
+  const messageWithLowercaseUUIDs = getMessageWithLowercaseUUIDs()
+
+  const translatedMessage = translator.convertFhirMessageToHl7V3ParentPrescriptionMessage(messageWithLowercaseUUIDs)
+
+  const allNonUpperCaseUUIDS = getAllUUIDsNotUpperCase(translatedMessage)
+  expect(allNonUpperCaseUUIDS.size).toBe(0)
+})
+
+function getMessageWithLowercaseUUIDs() {
+  const re = /[A-F0-9]{8}-[A-F0-9]{4}-[A-F0-9]{4}-[A-F0-9]{4}-[A-F0-9]{12}/g
+  let messageStr = LosslessJson.stringify(TestResources.examplePrescription1.fhirMessageUnsigned)
+  messageStr = messageStr.replace(re, (x) => x.toLowerCase())
```

Comment:
could you use a more descriptive variable name here than `x`? maybe something like `textMatch`?